src/Functional.Unions.FluentAssertions/Functional.Unions.FluentAssertions/UnionValueTypeAssertions.cs

```diff
@@ -0,0 +1,772 @@
+﻿using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+using Functional.Unions.FluentAssertions.Extensions;
+using System;
+using static Functional.Unions.FluentAssertions.Extensions.UnionExtensions;
+
+namespace Functional.Unions.FluentAssertions
+{
+	public class UnionValueTypeAssertions<TUnionType, TUnionDefinition, TOne, TTwo>
```

Comment:
Split this into multiple files as well?  I'm OK with `UnionValueTypeAssertions2.cs`, `UnionValueTypeAssertions3.cs`, etc.

Reply:
Will do, thanks for the suggestions!